Python/PrachiRungta/task2/storesqlite.py

```diff
@@ -0,0 +1,29 @@
+import csv
+import sqlite3
+
+db_filename='bookstore.db'
+data_filename='output/filename.csv'
+conn=sqlite3.connect(db_filename)
+
+cursor=conn.cursor()
+
+#create table
+cursor.execute('''create table if not exists book_stock
```

Comment:
You can use context manager here 
with con:
    con.execute(
Reference:
https://docs.python.org/2/library/sqlite3.html#using-the-connection-as-a-context-manager